marketplace_builder/views/pages/tutorials/variables/context.liquid

```diff
@@ -0,0 +1,112 @@
+---
+converter: markdown
+metadata:
+  title: The context variable
+  description: This guide will walk you thorugh the details of the context variable
+slug: tutorials/variables/context
+searchable: true
+---
+
+{{ page.metadata.description }}
+
+## Requirements
+
+To follow the steps in this tutorial, you should be familiar with:
+ * [Liquid Template language](https://shopify.github.io/liquid/)
+
+## Steps
+
+To learn everything about `context` variable please follow this three easy steps:
+1. {% assign step_1 = "What is context variable and how to use it." %}[{{ step_1 }}](#step-1)
+2. {% assign step_2 = "Context variable components" %}[{{ step_2 }}](#step-2)
+3. {% assign step_3 = "Expose local variable within context variable" %}[{{ step_3 }}](#step-3)
+
+### <a name="step-1" />Step 1: {{ step_1 }}
+
+Context is a global variable that accessible in every liquid file of the project including views, authorization policies and notifications files.
+To insepct it's structure and state, you can display it on the page as in the example below:
+
+{% raw %}
+```liquid
+---
+slug: context_example
+---
+{{ context }}
+```
+{% endraw %}
+
+### <a name="step-2" />Step 2: {{ step_2 }}
+
+{% raw %}
+* **context.authenticity_token** is created by the platform and automatically included in every form generated by `{% form %}` tag. It is used to mitigate [Cross Site Request Forgery](https://en.wikipedia.org/wiki/Cross-site_request_forgery) attacks. Has to be added to the payload in all platform requests other than GET.
+{% endraw %}
+
+* **context.session** object allows for quick access to session storage. Use mutations in order to store data in session as in [example](https://documentation.platform-os.com/api-reference/graphql/graphql-glossary#operation-name)
+* **context.current_user** object returns basic data of currently logged in user. Available attributes are: `first_name`, `last_name`, `email`, `id` and `slug`.
```

Comment:
```suggestion
* **context.current_user** object returns basic data of currently logged in user. Available attributes are: `first_name`, `last_name`, `email`, `id` and `slug`. It will return `nil` instead of object if user is not logged in.
```